### PR TITLE
Accept usage of Flask Blueprints as app instance

### PR DESCRIPTION
**Pull request by maryum375**
###  Summary

Allowing usage of slack blueprints to setup the SlackServer instance.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/python-slack-events-api/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Comment by jlapenna** [comment]
This is something I came across as well which would be nice to merge.

File: slackeventsapi/server.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def __init__(self, signing_secret, endpoint, emitter, server):
         # If a server is passed in, bind the event handler routes to it,
         # otherwise create a new Flask instance.
         if server:
-            if isinstance(server, Flask):
+            if isinstance(server, Flask) or isinstance(server, Blueprint):
                 self.bind_route(server)
             else:
                 raise TypeError("Server must be an instance of Flask")
```
Comment (psykzz) [inline slackeventsapi/server.py]:
```suggestion
                raise TypeError("Server must be an instance of Flask or Blueprint")
```

**Comment by seratch** [comment]
@maryum375 Thank you very much for taking the time to make this pull request. Also, I'm sorry that the team took a very long time (almost one year) until checking it.

I'm moving this forward. I've applied some additional changes to your PR in #69. It would be greatly appreciated if you check it and give us feedback if you have some.

**Comment by psykzz** [comment]
Any reason that needed to go into a new PR and couldn't have been added to this one? 
Typically PRs allow repo contributors push access to the branch, so it would've been possible.

If not it would be good to add @maryum375 to the co-authors via `Co-authored-by: Ran Mariuma <maryum375@gmail.com>`

**Comment by seratch** [comment]
@psykzz 
>Any reason that needed to go into a new PR and couldn't have been added to this one?
Typically PRs allow repo contributors push access to the branch, so it would've been possible.

Good point. Indeed, it's usually possible but somehow this repository seems to be a bit different. I've walked through the repository settings but haven't found any relevant settings to modify. So, I know it's not ideal but went with a new pull request.

> Co-authored-by

Thanks, I didn't know this! I don't think we need to modify my PR a lot but I'll update my commits with this.

**Comment by psykzz** [comment]
>  I've walked through the repository settings but haven't found any relevant settings to modify

Its a setting the PR author ticks on the PR to give access, typically i've found its enabled by default, Not in this case though, that's my mistake.
See the screenshot below 
image

Good to hear its included, mostly just curious about the flow.

**Comment by seratch** [comment]
@maryum375 Thanks for your contribution here. I just shipped 2.2.0rc1, a pre-release version including a fix for this. Please try it out if you have a chance. v2.2.0 will be out soon.

**Pull request closed without merging**